server/defaultConfig.json

```diff
@@ -1,7 +1,7 @@
 {
 	"version": 0.5,
 	"extractor": {
-		"pdf": "pdf2json",
+		"pdf": "pdfminer",
```

Comment:
please sync this changes in the Vue store also!

Reply:
done